README.md

```diff
@@ -4,7 +4,7 @@ OpenTelemetry community content.
 
 ## Get Involved
 
-There is a lot to do! If you are interested in getting involved, please join the mailing lists and attend the community meetings. If you wish to join one of the special interest groups, you can get in touch via [this simple form](https://forms.gle/HCGGoLRzoPm6o29K6). We are a friendly, collaborative group and look forward to working together.
```

Comment:
Minor point, but I kinda liked the "we are a friendly <blah blah blah>" part of the previous message, or at least the spirit of it. Generally I just want to make it psychologically easy for people without a social connection to existing OpenTel community members to get involved.